### ISSUE TITLE
Style/BracesAroundHashParameters auto correction can remove keys

### ISSUE THREAD

**Issue by DMA57361** [rubocop-hq/rubocop#6510]
I'm running corrections over some older code that hasn't been looked after by Rubocop before, and I've just had the Style/BracesAroundHashParameters cop auto-correct a hash that was laid out in a pretty ugly way and it has unexpectedly removed the last key.

The hash affected was effectively
```ruby
  HashWithIndifferentAccess.new(
    {'key1' => 1,
     'key2' => 2}
  )
```

--------

## Expected behavior

Remove the braces and correct indentation.

## Actual behavior

In additional to the expected behaviour, `'key2' => 2` was entirely removed.

## Steps to reproduce the problem

Given this contrived `eg.rb` file:

```ruby
def make_a_hash
  HashWithIndifferentAccess.new(
    {'key1' => 1,
     'key2' => 2}
  )
end
```

Run Rubocop corrections:

```
$ bundle exec rubocop --safe-auto-correct eg.rb
Inspecting 1 file
C

Offenses:

eg.rb:3:5: C: [Corrected] Style/BracesAroundHashParameters: Redundant curly braces around a hash parameter.
    {'key1' => 1, ...
    ^^^^^^^^^^^^^
eg.rb:3:16: C: [Corrected] Style/TrailingCommaInArguments: Avoid comma after the last parameter of a method call.
    'key1' => 1,
               ^

1 file inspected, 2 offenses detected, 2 offenses corrected
```

The `eg.rb` file is left in this state:

```ruby
def make_a_hash
  HashWithIndifferentAccess.new(
    'key1' => 1
  )
end
```

## RuboCop version

```
$ bundle exec rubocop -V
0.60.0 (using Parser 2.5.3.0, running on ruby 2.3.7 x86_64-darwin18)
```

**Comment by koic** [comment]
Thanks for the feedback. This issue has been fixed by #6443. It will be resolved in the next release of RuboCop (> 0.60.0) .
